src/components/map/KmlService.js

```diff
@@ -62,31 +61,31 @@ goog.require('ngeo.fileService');
         // Test regex here: http://regex101.com/r/tF3vM0/9
         // List of google icons: http://www.lass.it/Web/viewer.aspx?id=4
         kml = kml.replace(
-          /<href>http(?!(s:\/\/maps\.(google|gstatic)\.com[a-zA-Z\d\.\-\/_]*\.png|s?:\/\/[a-z\d\.\-]*(bgdi|geo.admin)\.ch))/g,
-          '<href>' + gaGlobalOptions.proxyUrl + 'http'
+            /<href>http(?!(s:\/\/maps\.(google|gstatic)\.com[a-zA-Z\d.\-/_]*\.png|s?:\/\/[a-z\d.-]*(bgdi|geo.admin)\.ch))/g,
```

Comment:
The above `[a-z\d.-]*` is matches exactly the same as  `.*`, which I'm not really a fain. I prefer as strict as possible - so really force literal point.